spec/std/isa/inst/Zvqdotq/vqdotus.vx.yaml

```
@@ -0,0 +1,55 @@
+﻿# Copyright (c) Kallal Mukherjee.
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+
+# yaml-language-server: $schema=../../../../schemas/inst_schema.json
+
+$schema: "inst_schema.json#"
+kind: instruction
+name: vqdotus.vx
+long_name: Vector 8-bit Unsigned-Signed Dot Product (vector-scalar)
+description: |
+  Vector quad widening unsigned-signed dot product instruction performing the dot product between a 4-element vector of 8-bit unsigned integer elements (vs2) and a scalar 4-element vector of 8-bit signed integer elements (xs1), accumulating the result into a 32-bit signed integer accumulator.
+
+  This instruction is only defined for SEW=32. It works on an element group with four 8-bit values stored together in a 32-bit bundle. For each input bundle for the dot product there is a corresponding (same index) SEW-wide element in the accumulator source (and destination).
+
+  The "q" in the mnemonic indicates that the instruction is quad-widening. The number of body bundles is determined by `vl`. The operation can be masked, each mask bit determines whether the corresponding element result is active or not.
+
+  The operation performed is:
+  ```
+  vd[i] = vs2[i][0] * xs1[0] + vs2[i][1] * xs1[1] + vs2[i][2] * xs1[2] + vs2[i][3] * xs1[3] + vd[i]
+  ```
+
+  Where vs2[i] is a 32-bit bundle containing four 8-bit unsigned integers and xs1 contains four 8-bit signed integers in its lower 32 bits.
+definedBy: Zvqdotq
+assembly: vd, vs2, xs1, vm
+encoding:
+  match: 101001-----------110-----1010111
```

Nicely, the CI detected that this is not correct.
```suggestion
  match: 101110-----------110-----1010111
```

---

This still needs to be fixed.